packages/cadl-ranch-api/src/types.ts

```diff
@@ -6,6 +6,7 @@ import { MockRequest } from "./mock-request.js";
  */
 export interface RequestExt extends Request {
   rawBody?: string;
+  files?: any;
```

Comment:
any better type for this? I see below it should be an array with some expected properties

Reply:
added type for `files`